<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>I - Update</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="comboBoxMarkTypeSlct"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUpdateMrks">
            <property name="text">
             <string>Update </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableSubjecInfo">
          <property name="styleSheet">
           <string notr="true">QHeaderView {
	
	background-color: rgb(230, 230, 230);
border-radius: 10px;
}

QHeaderView::section
{
spacing: 40px;

border: 1px solid gray;

padding: 4px;

border-radius: 6px;

background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(223, 210, 170, 12), stop:1 rgba(255, 255, 255, 255));

margin: 5px;

text-align: centre;
	
font: 75 10pt &quot;Trebuchet MS&quot;;

}

QTableView::item
{
  border: 0px;
  padding: 5px;
}
</string>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::NoPen</enum>
          </property>
          <property name="rowCount">
           <number>4</number>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>50</number>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelProgress">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarWebInd">
            <property name="value">
             <number>24</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="itsBrowser" name="wFitsbrowser" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuSubject">
    <property name="title">
     <string>Subject</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuIEnabler">
    <property name="title">
     <string>iEnabler</string>
    </property>
    <addaction name="actionMake_empty_0"/>
    <addaction name="actionMake_all_0"/>
    <addaction name="separator"/>
    <addaction name="actionBlackout_Cancelled"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionLoad_ec_tut_ac_za"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuSubject"/>
   <addaction name="menuIEnabler"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load </string>
   </property>
  </action>
  <action name="actionMake_all_0">
   <property name="text">
    <string>Make all 0</string>
   </property>
  </action>
  <action name="actionMake_empty_0">
   <property name="text">
    <string>Make empty 0</string>
   </property>
  </action>
  <action name="actionBlackout_Cancelled">
   <property name="text">
    <string>Blackout Cancelled</string>
   </property>
  </action>
  <action name="actionLoad_ec_tut_ac_za">
   <property name="text">
    <string>Load ec.tut.ac.za</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>itsBrowser</class>
   <extends>QWidget</extends>
   <header>itsbrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
